libfreerdp/core/nla.c

```diff
@@ -99,6 +99,53 @@
 
 #define TERMSRV_SPN_PREFIX	"TERMSRV/"
 
+
+struct rdp_nla
+{
+	BOOL server;
+	NLA_STATE state;
+	int sendSeqNum;
+	int recvSeqNum;
+	freerdp* instance;
+	CtxtHandle context;
+	LPTSTR SspiModule;
+	char* SamFile;
+	rdpSettings* settings;
+	rdpTransport* transport;
+	UINT32 cbMaxToken;
+#if defined(UNICODE)
+	SEC_WCHAR* packageName;
+#else
+	SEC_CHAR* packageName;
+#endif
```

Comment:
This is what TCHAR is for.

Reply:
Correct, did not look at the content of the struct, just moved it